From eb299e3faaf2447ba2fc13193eef9fc6c0efd83b Mon Sep 17 00:00:00 2001
From: Mohsen Dolaty <mohsen.dolaty@amd.com>
Date: Fri, 22 Sep 2023 12:06:30 -0500
Subject: [PATCH 1/1] linux-speed: Remove debug prints from i3c driver

Remove extra debug prints from i3c driver.
change pr_info to pr_debug

Signed-off-by: Mohsen Dolaty <mohsen.dolaty@amd.com>
---
 drivers/i3c/master.c               | 2 +-
 drivers/i3c/master/dw-i3c-master.c | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/i3c/master.c b/drivers/i3c/master.c
index 4e7aacc3c7d3..6b5fb95f3c4f 100644
--- a/drivers/i3c/master.c
+++ b/drivers/i3c/master.c
@@ -1524,7 +1524,7 @@ static int i3c_master_pre_assign_dyn_addr(struct i3c_dev_desc *dev)
 		if(master->set_dasa) {
 			if(dev->info.static_addr !=  I3C_MUX_SA) { // Do Not issue SET DASA for I3C Mux
 				i3c_master_setdasa_locked(master, dev->info.static_addr, dev->boardinfo->init_dyn_addr);
-				pr_info( "i3c_master_pre_assign_dyn_addr: Reattach Bus %x SA %x to DA %x\n",
+				pr_debug( "i3c_master_pre_assign_dyn_addr: Reattach Bus %x SA %x to DA %x\n",
 				master->bus.id, dev->info.static_addr, dev->boardinfo->init_dyn_addr );
 			}
 		}
diff --git a/drivers/i3c/master/dw-i3c-master.c b/drivers/i3c/master/dw-i3c-master.c
index d53ed346deb5..e56b247f7e96 100644
--- a/drivers/i3c/master/dw-i3c-master.c
+++ b/drivers/i3c/master/dw-i3c-master.c
@@ -1009,7 +1009,8 @@ static int dw_i3c_master_send_ccc_cmd(struct i3c_master_controller *m,
 		ret = dw_i3c_ccc_get(master, ccc);
 	else
 		ret = dw_i3c_ccc_set(master, ccc);
-	pr_info( "dw_i3c_master_send_ccc_cmd: " "bus %d, cmd 0x%x r/w %d ret %d\n",m->bus.id, ccc->id , ccc->rnw, ret);
+	if(ret != 0)
+		pr_err( "dw_i3c_master_send_ccc_cmd: Error bus %d, cmd 0x%x r/w %d ret %d\n",m->bus.id, ccc->id , ccc->rnw, ret);
 	if ((ccc->id == I3C_CCC_SETAASA) || (ccc->id == I3C_CCC_SETHID)) {
 		writel(i3c_pp_timing, master->regs + SCL_I3C_PP_TIMING);
 	}
--
2.25.1
